Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
```

Comment:
I think it would be desirable to compile the unit tests the same way we compile the rest (see another comment). In case a unit test fails, a developer can recompile the project using a debug build in which case the line information should be correct.